src/components/button.js

```diff
@@ -0,0 +1,46 @@
+import React,  { Component, } from "react";
+import PropTypes from "prop-types";
+import { Button, View, TouchableOpacity, Text} from 'react-native';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247201)
```suggestion
import { Button, View, TouchableOpacity, Text} from "react-native";
```